content/en/docs/concepts/storage/volumes.md

```diff
@@ -996,18 +996,22 @@ For more information, see the [vSphere volume](https://github.com/kubernetes/exa
 
 {{< feature-state for_k8s_version="v1.19" state="beta" >}}
 
-The `CSIMigration` feature for `vsphereVolume`, when enabled, redirects all plugin operations
-from the existing in-tree plugin to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver. In order to use this feature, the
+The `CSIMigration` feature for `vsphereVolume`,  is enabled by default as of Kubernetes v1.25.
+All plugin operations from the in-tree `vspherevolume` will be redirected to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver. 
```

Comment:
```suggestion
All plugin operations from the in-tree `vsphereVolume` will be redirected to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver. 
```

Reply:
This is only true provided that you don't force `CSIMigration` to off, and you also don't force `CSIMigrationvSphere` to off   (or maybe your cloud provided made that choice for you) - and we should make that clear.

Reply:
both `CSIMigration` and `CSIMigrationvSphere` are enabled in K8s 1.25, so do we need to mention that we can disable `CSIMigration` and `CSIMigrationvSphere` features?

Reply:
@sftim how about following message

> All plugin operations from the in-tree `vspherevolume` will be redirected to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver unless feature gates `CSIMigration` or `CSIMigrationvSphere` are disabled.